packages/x-charts-pro/src/ChartsToolbarPro/internal/ChartsToolbarZoomInButton.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+
+import * as React from 'react';
+import { ToolbarButton } from '@mui/x-charts/Toolbar';
+import { useChartContext } from '@mui/x-charts/internals';
+import materialSlots, { type ChartsSlots } from '@mui/x-charts/material';
+import { useChartsLocalization } from '@mui/x-charts/hooks';
+import { UseChartProZoomSignature } from '../../internals/plugins/useChartProZoom';
+
+type ChartsToolbarZoomInButtonSlots = Partial<Pick<ChartsSlots, 'baseTooltip' | 'zoomInIcon'>>;
+
+type ChartsToolbarZoomInButtonSlotProps = {
+  [K in keyof Required<ChartsToolbarZoomInButtonSlots>]: React.ComponentProps<
+    Required<ChartsToolbarZoomInButtonSlots>[K]
+  >;
+};
+
+interface ChartsToolbarZoomInButtonProps {
+  slots?: ChartsToolbarZoomInButtonSlots;
+  slotProps?: ChartsToolbarZoomInButtonSlotProps;
+}
+
+export function ChartsToolbarZoomInButton({ slots, slotProps }: ChartsToolbarZoomInButtonProps) {
```

Comment:
Not sure we need these two buttons 🤔 

Though it seems we need different logic for them, so it might make sense.

Reply:
I think we need both buttons so that they can be customized. I'll add the `render` prop in a follow-up PR.